lms/envs/common.py

```diff
@@ -1620,6 +1620,8 @@
     'survey',
 
     'lms.djangoapps.lms_xblock',
+
+    'openedx.core.djangoapps.content.course_structures',
```

Comment:
Hmm - naming conventions have yet to be proposed in `openedx.core.djangoapps`. What's in there now are `course_groups` and `user_api` directories, which seems a bit arbitrary. Is it worth sketching out the structure we'd like to see there?